src/elements/components/model-card.module.scss

```diff
@@ -48,6 +53,17 @@
         }
     }
 
+    .paired {
+        position: absolute;
+
+        backdrop-filter: blur(2rem) saturate(1.5) opacity(0.95);
```

Comment:
1. Why the `opacity(0.95)`? This mixes the original images with the blurred and saturated image for a weird effect.
2. I think we can go up to `saturate(2)`.